src/kvstore/RocksEngine.cpp

```diff
@@ -459,9 +665,14 @@ nebula::cpp2::ErrorCode RocksEngine::flush() {
   rocksdb::FlushOptions options;
```

Comment:
in fact, I think flush() should be some separated functions. i.e., flush(all/default) flush(data) flush(ref). They will be operated separately.

but it is ok now.